### ISSUE TITLE
Promoting RChain on Facebook

### ISSUE THREAD

**Issue by qelentium** [rchain/bounties#791]
### Benefit to RChain

Promoting RChain on Facebook.My page have 3700 users there.Most of them are developers,designers or blockchain enthusiasts.I will write 3 articles about RChain: 1.Introduce to RChain + How to join Coop.  2.Technology + How to become developer.  3. Conclusion and why RChain is the best choice to invest.

### Budget and Objective

*Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely.*

_Estimated Budget of Task:_ **$[2000]**
_Estimated Timeline Required to Complete the Task:_ **[1 week]**
_How will we measure completion?_ **[Every post gonna be opened so everyone can see it and links gonna be merged here]**

*See CONTRIBUTING.md for details on budget and reward process.*

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by qelentium** [comment]
![proof](https://user-images.githubusercontent.com/37076124/41820288-054d44f2-77d8-11e8-86de-56c876542eef.jpg)
Proof of real number of friends

**Comment by qelentium** [comment]
[Posted video about RChain](https://www.facebook.com/qelentium/posts/921135521406903)

**Comment by deannald** [comment]
@qelentium These are shaping up to be nice introductory blog posts. It would be nice to use them outside of your Facebook page. Perhaps we can add them to the RChain blog or a community led blog? @PatrickM727 @kitblake

**Comment by qelentium** [comment]
Yeah, why not. Thanks for responding

**Comment by AyAyRon-P** [comment]
Help me understand why there is $2000 in value to RChain by you making 2 posts to your personal page? Both of these posts seem to be plagiarized and reorganized content mixed between these two links. 

Your first post content appears to be a direct reorganization of this:
https://www.weusecoins.com/what-is-rchain/

The first paragraph of your second post is a direct line by line copy from: 
https://www.geekwire.com/2017/seattle-based-rchain-takes-bitcoin-ethereum-new-blockchain-tech/
and also has pieces of content from the weusecoins.com link

@kitblake @pmoorman @deannald @PatrickM727

**Comment by qelentium** [comment]
@AyAyRon-P its gonna be 3 posts (the third one is coming today).The budget came from audience i have on my page, plus researching and gathering information of what is RChain by personal understanding and other resources.I took some information from resources that you mentioned.

**Comment by kitblake** [comment]
@qelentium Imho Facebook posts have a severely limited value because devs, who are the audience we're trying to reach, don't hangout on Facebook. Having 3631 friends is no guarantee of reach. Plus it's a closed platform, every time I tap a Facebook link it asks me to login so I don't do it anymore. If you do login then Facebook displays you how many people reacted, so I guess you could report that.

Have you considered posting on Medium? It's an open forum, popular with devs and crypto people alike, and posts have a measure of impact in the number of claps they receive. Then I'd be much more inclined to assign some budget for effort while we see how the posts are received.

**Comment by deannald** [comment]
I agree with @kitblake - cross-posting to other sites would help with reach. As with translations, blog posts and other articles, we should be looking at the number of views, reactions, etc. to determine reach. Original content (by this I mean, not copied from other sources) should be rewarded through the bounty system, but only if that content is placed somewhere where people are seeing it. It doesn't do RChain as much good to pay out for this kind of work if no one is looking at it.

**Comment by qelentium** [comment]
@deannald agreed with statement if nobody see it - there is no point. As i said i will post it in different resources and merge links here too.

**Comment by jplepel** [comment]
@qelentium Can you share some more of the analytics from your posts? You have friends but looks like engagement is low.

**Comment by qelentium** [comment]
@jplepel this is true, i dont have much likes or reacts to this posts. I'm posting them  now at medium  and will share links here

**Comment by AyAyRon-P** [comment]
You said “I took some information from resources that you mentioned.” It’s not a resource when it’s a direct copy and paste. This information is readily available on the internet as it is, there is no originality in this content. 

I see zero justification for a $2000 bounty just because you have 3600 friends on Facebook who may not even be our target audience. More importantly, the only metric for views in the number of likes you get on each post. 

At best I see this as a few hours of text editing, we're not in the business of selling “likes” on Facebook.

**Comment by PatrickM727** [comment]
@qelentium it'd be nice to post these articles to our blog or guest post them to other major publications first. are you up for that?

**Comment by qelentium** [comment]
@PatrickM727 why not

**Comment by PatrickM727** [comment]
Unless they are copies. 

@pmoorman @AyAyRon-P can determine this

**Comment by qelentium** [comment]
@AyAyron-P said they are, so i guess not

**Comment by AyAyRon-P** [comment]
If you feel like I'm wrong in calling attention to this, please say so, based on the facts of what I'm seeing there is no original content here. Unless you're the author of the articles on weusecoins.com and captainaltcoins.com.  

@pmoorman @PatrickM727 

![2](https://user-images.githubusercontent.com/35570702/42171841-fae4dfc6-7ddf-11e8-910e-2f11afb94051.PNG)
![3](https://user-images.githubusercontent.com/35570702/42171842-faf9a50a-7ddf-11e8-9cf5-22d8c95e5299.PNG)
![1](https://user-images.githubusercontent.com/35570702/42171843-fb0aa2c4-7ddf-11e8-9aa1-192d94427f43.PNG)
![1b](https://user-images.githubusercontent.com/35570702/42172135-ca8d7e0e-7de0-11e8-86de-445b2c3b5d86.PNG)
![1c](https://user-images.githubusercontent.com/35570702/42172136-caa0f678-7de0-11e8-9c30-d59586152ab7.PNG)
![1d](https://user-images.githubusercontent.com/35570702/42172137-cab1ab94-7de0-11e8-9241-70bbd52141c6.PNG)
![1e](https://user-images.githubusercontent.com/35570702/42172138-cac7a26e-7de0-11e8-977f-d72389cc9b22.PNG)

**Comment by PatrickM727** [comment]
^ that's hard to deny

I'm liking this guide system already
